src/p11_attribute.ml

```diff
@@ -167,9 +162,9 @@ let to_json : type a . a t -> Yojson.Safe.json = fun attribute ->
     | CKA_MODIFIABLE, param ->
         p_bool "CKA_MODIFIABLE" param
     | CKA_EC_PARAMS, param ->
-        p_ec_params "CKA_EC_PARAMS" param
+        p_data "CKA_EC_PARAMS" param
```

Comment:
What should be tested exactly?  The fact that it uses `p_data` or the fact the `p_data` serializes its second argument to hex?